moonstone/normalization/reads/read_downsize.py

```diff
@@ -0,0 +1,69 @@
+import logging
+import random
+
+from moonstone.normalization.reads.base import BaseDownsizing
+
+logger = logging.getLogger(__name__)
+
+
+class DownsizePair(BaseDownsizing):
+    """Normalization for the purposes of assessing diversity. Reads are downsized by random selection of raw reads
+    generating a subset from which alpha diversity can be calculated.
+    Note that removal of data, while useful for diversity assessment, is no longer considered good practice.
+    https://doi.org/10.1371/journal.pcbi.1003531
+    """
+
+    def __init__(self, raw_f, raw_r, n=1000, seed=623):
+        """Paired reads assumes forward and reverse FASTQ files.
+        n is the number of reads that will be randomly picked, with a default of 1000.
+        A random seed is preset to 62375 to allow for reproducibility"""
+
+        super().__init__(raw_f, raw_r)
+        self.raw_f = raw_f
+        self.raw_r = raw_r
+        self.downsize_to = n
+        self.seed = seed
+
+    def downsize_pair(self):
+        if self.raw_f == self.raw_r:
+            print(f"\nWarning files {self.raw_f} and {self.raw_r} are the same! Expected Forward and Reverse!\n")
+
+        records: int = sum(1 for _ in open(self.raw_f)) // 4
+        print('Found %i reads' % records)
```

Comment:
Maybe you could use the logger.info here?

Reply:
Yes. The print statements were for testing purposes, and are not logger entries.